internal/pkg/runtime/engines/imgbuild/engine_linux.go

```diff
@@ -91,6 +92,13 @@ func (e *EngineOperations) PrepareConfig(starterConfig *starter.Config) error {
 		starterConfig.SetHybridWorkflow(true)
 		starterConfig.SetAllowSetgroups(true)
 
+		starterConfig.SetTargetUID(0)
```

Comment:
A few minor modifications to avoid some code duplication (I am being picky)... see following suggestions.

Reply:
As discussed on Slack, putting `starterConfig.SetTargetGID([]int{0})` outside of those two conditions could fail later in C starter code when calling `setgroups` if for some reasons setgroups usage is denied